community/clinics/index.shtml

```diff
@@ -22,26 +22,37 @@
     
     <p>Upcoming JMRI Clinics as submitted or found on the web:
 
-    July 3-10, 2016 - Indianapolis, IN
```

Comment:
Not related to this PR, but these clinics are not upcoming. It seems reasonable to move them to the next section: Past clinics.